### PR TITLE
Milestone 3: Complete some details

### PR DESCRIPTION
**Pull request by eroiyo**
for this milestone we added some styles to the project!

-Specials color for the first 3 places in the board
 *yellow, silver and brown respectively

-Green Buttons

-Mostly 2 tones of dark brown for the background

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @eroiyo,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._
- [ ] Update the readme with the live link of the project.
- [ ] The screenshot of the project doesn't appear on the readme. Kindly fix this too.
- Please refer to other comments on the code.

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 1]
```diff
@@ -13,16 +13,30 @@ const scoreInput = document.getElementById('score-input');
...
+function ShowOne(object, placenumber) {
   const div = document.createElement('div');
-  div.textContent = `${object.user}: ${object.score}`;
+  const showname = document.createElement('h3');
+  const shownumber = document.createElement('h3');
+  const place = document.createElement('h3');
+
+  showname.textContent = object.user;
+  shownumber.textContent = object.score;
+  place.textContent = `#${placenumber}`;
```
Comment (mikenath223) [inline src/index.js:24]: - [ ] Kindly replace all `regular` functions in `form.js` and `index.js` modules with `arrow` functions. In the [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code.) it's required to use es6 syntax.

Comment (eroiyo) [reply src/index.js:24]: Thanks, I will do the corrections.

File: src/form.js [hunk 2]
```diff
@@ -12,24 +12,24 @@ function send(e, nameInput, scoreInput) {
...
+const textLimit = (input) => {
   if (input.value.length > 30) {
     alert('the name is too long');
     return false;
   }
   return true;
-}
+};
 
 export default function submitOrNota(event, scoreInput, nameInput) {
```
Comment (vichuge) [inline src/form.js:34]: - [ ] It looks like we have a function without the arrow way yet, please work on it. 🐶
- [ ] On this same script on line 9, there's another function that needs to be an arrow function too. Please change it. 🐱
image

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ♻️ ⚠️ ⚠️ 🚦 🚦 

Hi dear student, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

star_wars_almost_there


## Aspects to improve ♻️
_Check the comments under the review._

- [ ] Check the comments under the review. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/form.js (hunk 2 above)
Comment (eroiyo) [reply src/form.js:34]: Thanks i will work in it

**Review by hemant-soni-vst-au4 — APPROVED** [review]
Hi @eroiyo, 

Good work on making the requested changes. 🚀 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eroiyo** [comment]
Thanks

**Pull request merged**